### PR TITLE
Expose gettransactiondetails

### PR DESCRIPTION
**Pull request by jgriffiths**
For fetching non-wallet transactions

### CODE REVIEW

File: green_cli/common.py [hunk 1]
```diff
@@ -389,6 +389,15 @@ def _txlist_summary(txlist):
...
     return '\n'.join(lines)
 
+@green.command()
+@click.argument('txid', type=str)
+@with_login
+def gettransactiondetails(session, txid):
+    """Get transaction details of an arbitrary transaction."""
+    result = gdk.get_transaction_details(session.session_obj, txid)
+    result = format_output(json.loads(result))
+    click.echo(result)
```
Comment (jkauffman1) [inline green_cli/common.py:399]: Should just be able to use the `@print_result` decorator?

Comment (jgriffiths) [reply green_cli/common.py]: Updated, ta

Comment (jkauffman1) [reply green_cli/common.py]: sorry to nit again but I think the best form for this is:

`return session.get_transaction_details(txid)`

see e.g. `getfeeestimates`

**Comment by jkauffman1** [comment]
ack b164236e6a43e2113deea7ef6be6145d25bfc327, with nit

**Pull request merged**